.travis.yml

```diff
@@ -42,6 +52,27 @@ before_script:
       dbus-launch /usr/bin/python -c \
         "import gnomekeyring;gnomekeyring.create_sync('login', '');"
     fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then bash ./script/setup-macos-keychain; fi
```

Comment:
I'd also suggest checking this isn't run for PRs initiated from forks

Reply:
Great catch! Added check for secure variables which tells us whether we're on a fork!